lib/chef_zero/server.rb

```diff
@@ -103,14 +104,15 @@ class Server
       '/version',
     ]
 
-    def initialize(options = {})
-      @options = DEFAULT_OPTIONS.merge(options)
-      if @options[:single_org] && !@options.has_key?(:osc_compat)
-        @options[:osc_compat] = true
+    def initialize(**options)
```

Comment:
`*options` means it will grab all the items passed and make it a list while `**options` means it will grab all the items passed and make them a hash